server/src/fixed-asset/dto/asset-response.dto.ts

```diff
@@ -0,0 +1,64 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { Expose } from 'class-transformer';
+
+export class AssetResponseDto {
+  @ApiProperty({ example: '65f1c7a4e52891827ad41234' })
+  @Expose()
```

Comment:
Kindly ask you to remove that Expose annotation for all the fields since all of them can be exposed by default with class-transformer.